mwclient/client.py

```diff
@@ -329,9 +327,10 @@ def api(self, action, http_method='POST', *args, **kwargs):
             >>> result = site.api('query', prop='coordinates', titles='Oslo|Copenhagen')
             >>> for page in result['query']['pages'].values():
             ...     if 'coordinates' in page:
-            ...         print('{} {} {}'.format(page['title'],
-            ...             page['coordinates'][0]['lat'],
-            ...             page['coordinates'][0]['lon']))
+            ...         title = page['title']
+            ...         lat = page['coordinates'][0]['lat']
+            ...         lon = page['coordinates'][0]['lon']
+            ...         print(f'{title} {lat} {lon}')
```

Comment:
jesus this thing is smart

Reply:
Not that smart sadly, that one was me grepping manually 😅